NoClassDefFoundError when trying to create a topic with latest version

After adding #125 and #126 the `NoClassDefFoundError` error is always happening when trying to create a topic with the following command:
```
topic create --bootstrap-server=XXXX --topic=topicName --topic-partitions=N --topic-rep-factor=M
```

```
Exception in thread "main" java.lang.NoClassDefFoundError: com/fasterxml/jackson/annotation/JsonView
	at com.fasterxml.jackson.databind.introspect.JacksonAnnotationIntrospector.<clinit>(JacksonAnnotationIntrospector.java:37)
	at com.fasterxml.jackson.databind.ObjectMapper.<clinit>(ObjectMapper.java:400)
	at io.fabric8.kubernetes.client.utils.KubernetesSerialization.<init>(KubernetesSerialization.java:76)
	at io.fabric8.kubernetes.client.KubernetesClientBuilder.<init>(KubernetesClientBuilder.java:51)
	at io.strimzi.kafka.AbstractKubernetesConfigProvider.configure(AbstractKubernetesConfigProvider.java:63)
	at io.strimzi.kafka.KubernetesConfigMapConfigProvider.configure(KubernetesConfigMapConfigProvider.java:17)
	at org.apache.kafka.common.config.AbstractConfig.instantiateConfigProviders(AbstractConfig.java:620)
	at org.apache.kafka.common.config.AbstractConfig.resolveConfigVariables(AbstractConfig.java:540)
	at org.apache.kafka.common.config.AbstractConfig.<init>(AbstractConfig.java:114)
	at org.apache.kafka.common.config.AbstractConfig.<init>(AbstractConfig.java:146)
	at org.apache.kafka.clients.admin.AdminClientConfig.<init>(AdminClientConfig.java:298)
	at org.apache.kafka.clients.admin.Admin.create(Admin.java:133)
	at io.strimzi.arguments.topic.CreateTopicCommand.createTopics(CreateTopicCommand.java:50)
	at io.strimzi.arguments.topic.CreateTopicCommand.call(CreateTopicCommand.java:42)
	at io.strimzi.arguments.topic.CreateTopicCommand.call(CreateTopicCommand.java:25)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
	at picocli.CommandLine.execute(CommandLine.java:2170)
	at io.strimzi.Main.main(Main.java:23)
Caused by: java.lang.ClassNotFoundException: com.fasterxml.jackson.annotation.JsonView
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	... 24 more
```

Kafka Version used: 4.0.0

--- a/admin/pom.xml
+++ b/admin/pom.xml
@@ -59,7 +59,6 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-annotations</artifactId>
-            <scope>test</scope>
         </dependency>
     </dependencies>
 

